packages/core/tests/test_api.py

```
@@ -0,0 +1,250 @@
+        payload = {
+            "name": "new-skill",
+            "description": "A newly created skill",
+            "template": "python"
+        }
+        response = test_client.post("/skills", json=payload)
+        assert response.status_code == 200
+        data = response.json()
+        assert data["status"] == "ok"
+        assert "result" in data
+    
+    def test_create_skill_duplicate(self, test_client, sample_skill_dir):
```

Ruff (0.14.10)

Unused method argument: `sample_skill_dir`

(ARG002)